src/main/java/frc/robot/Robot.java

```diff
@@ -92,4 +111,23 @@ public void testInit() {
   /** This function is called periodically during test mode. */
   @Override
   public void testPeriodic() {}
+
+
+  private void UpdateBotPose() {
+    Double[] defaultPos = new Double[]{0.0,0.0,0.0,0.0,0.0,0.0};
+    Double[] bp = NetworkTableInstance.getDefault().getTable("limelight").getEntry("botpose").getDoubleArray(defaultPos);
```

Comment:
Similar question here with regard to what will happen if `getTable("limelight")` fails.

Aside from the possibility during competition that our camera fails, I'm thinking about making sure we can deploy to test robots without one present.

Reply:
OK, I think I answered my question. All of the `get` calls create the item if it is missing.